ci/build/build-pipeline.yaml

```diff
@@ -510,7 +558,7 @@ spec:
       - in_parallel:
           steps:
           - get: chart
-            passed: [package]
+            passed: [delete]
```

Comment:
This is weird.... We're testing a package whose deployment has been deleted?
Or is this intentional? We're always deleting the test instance before testing?

Reply:
If the latter is the case - just add this task to the test job - no need for a new job here

Reply:
Yep, always deleting the previous test instance before recreating and testing. Ensures we're starting from a new release for all the components.

Reply:
Ok then the `task: delete-test-release` logically belongs to the test job - clean up before releasing a new test instance

Reply:
The plan was to have a separate job so we can re-use the logic. This will be triggered nightly, as well as before any test release. That way the test environment will be left after a test run (which may be useful for debugging), but will never live for more than 24 hours, to deal with the HSM key issue.

Reply:
All right then, I guess that makes sense